src/controllers/SubmissionSummaryController.ts

```diff
@@ -0,0 +1,14 @@
+import { controller, BaseHttpController, httpGet } from 'inversify-express-utils';
+import { SUBMISSION_SUMMARY_PAGE_URI } from '../utils/Paths';
+
+@controller(SUBMISSION_SUMMARY_PAGE_URI)
+export class SubmissionSummaryController extends BaseHttpController {
+
+    @httpGet('')
+    public renderView(): void {
+
+        const session = this.httpContext.request.session;
+
+        this.httpContext.response.render('submission-summary', {session});
```

Comment:
if there is no session, would we still want to pass this?

Reply:
Is it possible for a user to reach this page without a session? Wouldn't we push them back to the beginning if they tried to skip the journey? Surely if they do reach this stage with no info, the test should reflect that